.github/custom_python_actions/check_cla_issue.py

```diff
@@ -11,7 +11,7 @@ def main() -> None:
 
     gh = github3.login(token=gh_token)
     if not gh:
-        raise Exception("GH_TOKEN not correctly set")
+        raise Exception("github login failed - maybe GH_TOKEN was not correctly set")
```

Comment:
Why not check if gh_token is empty and raise exceptions with according message?

Reply:
I'm not able to see exactly what the output of the env var `GH_TOKEN` is - whether it's empty or jibberish.  If it weren't set at all it would throw a `KeyError` when trying to access it. The error that it throws is `'NoneType' object has no attribute 'organization'` so I figured it would be best to check if the `gh` object is empty.